server/src/utils/getConfig.test.ts

```diff
@@ -138,13 +141,16 @@ describe('config', () => {
       .spyOn(getDefaultConfig, 'getDefaultConfig')
       .mockImplementation((): any => mockDefaultConfig);
 
-    const config = getConfig.getServerConfig();
+    const serverConfig = getConfig.getServerConfig();
+    serverConfig.communicationServicesConnectionString = 'endpoint=test_endpoint_value;accesskey=secret';
+    const clientConfig = getConfig.getClientConfig(serverConfig);
```

Comment:
Hmm not sure why we needed to add this call here. The test is for `getServerConfig`

Reply:
That was my bad, forgot to revert from something I was trying. Fixed this now.